tests/queries/0_stateless/02677_hash_functions_on_null.sql

```diff
@@ -0,0 +1,15 @@
+select xxHash32(NULL);
```

Comment:
Cosmetic: could we please put SQL keywords in all-lowercase or all UPPERCASE but not a mix of both?